src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderMovementsResourceIntTest_moveToCell.kt

```diff
@@ -129,11 +129,11 @@ class OffenderMovementsResourceIntTest_moveToCell : ResourceTest() {
     val response = requestMoveToCell(
       createJwt("unilink", listOf("ROLE_UNILINK")),
       OFFENDER_NO,
-      INITIAL_CELL_DESC,
+      NEW_CELL_DESC,
```

Comment:
turns out my test was passing since it didn't then call `saveAndReturnCellMoveResult` so didn't go into the extra restrictions required.  Changed to actually move prisoner cell.